FirstProject/new-evaluation.html

```diff
@@ -85,39 +83,33 @@ <h3 style="color:white;">Senior</h3>
             </div>
         </div>
 
-        <div class="row">
-            <h2 class="description_section" style="color: #03A9F4;">Should the candidate be hired?</h2>
-            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="The type of project that is suitable for the candidate-->
-<!--Is guidance required for the candidate-->
-<!--" ></textarea>-->
-            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+        <div class="row_first">
+            <h2 class="description_section">Should the candidate be hired?</h2>
+            <textarea cols="164" placeholder="The type of project that is suitable for the candidate
+Is guidance required for the candidate
+" ></textarea>
 
-            </div>
         </div>
 
-        <div class="row">
-            <h2 class="description_section" style="color: #03A9F4;">General Impression</h2>
-            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="*required-->
-<!--" ></textarea>-->
-            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+        <div class="row_first">
+            <h2 class="description_section">General Impression</h2>
+            <textarea cols="164" placeholder="*required
+" ></textarea>
 
-            </div>
         </div>
 
-        <div class="row">
-            <h2 class="description_section" style="color: #03A9F4;">Workflow, Leadership & Soft Skills</h2>
-            <!--<textarea cols="194" style="margin-right: 282px; margin-left: 252px; height:150px; border: 2px solid grey;" placeholder="Describe the environment in which the candidate works.-->
-<!--Describe any guidance or management experience-->
-<!--" ></textarea>-->
-            <div class="#" style="width:62.2%; height:130px; border:2px solid grey; margin: 0 auto;">
+        <div class="row_first">
+            <h2 class="description_section">Workflow, Leadership & Soft Skills</h2>
```

Comment:
Unescaped HTML entity is potentially unsafe. Let's do an audit of all usages and fix this throughout.